src/components/CanvasEditor/Footer/Help/Help.scss

```diff
@@ -33,21 +34,41 @@
   }
 
   &__shortcuts {
-    border: 1px solid var(--light-border-color);
-    border-radius: var(--default-border-radius);
+    display: flex;
+    flex-direction: column;
 
     &-shortcut {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
       border-bottom: 1px solid var(--light-border-color);
 
       &:last-child {
         border-bottom: none;
       }
     }
 
+    &-content {
+      display: flex;
+      align-items: center;
+      width: 100%;
+
+    }
+    &-text {
+      display: flex;
+      align-items: center;
+      margin-right: 10px;
```

Comment:
I don't think so we need this margin-right  too ..

Reply:
ok 
I'm on it

Reply:
Also please take latest pull from master. Bit of things changed from initial release.
Thanks !

Reply:
sure i'll pull master to my local